### PR TITLE
[chore](CI)Regularly check project quality with Sonar

### PR DESCRIPTION
**Pull request by CalvinKirs**
# Proposed changes
Since the fork warehouse cannot obtain the token (for security reasons),
So set it as a timed check.
Issue Number: close #xxx

## Problem summary

Describe your changes.

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [ ] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: .github/workflows/sonarcloud.yml [hunk 1]
```diff
@@ -66,43 +50,16 @@ jobs:
           key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
           restore-keys: ${{ runner.os }}-m2
       - name: Setup thrift
-        if: steps.java-changes.outputs.any_changed == 'true'
         run: |
           sudo apt-get update
           sudo DEBIAN_FRONTEND=noninteractive apt-get install -y thrift-compiler=0.13.0-2build2
```
Comment (yangzhg) [inline .github/workflows/sonarcloud.yml:55]: thrift-0.16.0 ？

Comment (CalvinKirs) [reply .github/workflows/sonarcloud.yml]: yup, thanks

**Review by yangzhg — APPROVED** [review]
LGTM

**Comment by CalvinKirs** [comment]
run buildall

**Pull request merged**